### PR TITLE
chore: document linux only plugins (part 2)

### PR DESCRIPTION
**Pull request by powersj**
Adds a note to the config for linux-only or non-windows only plugins as well as any arch limitations.

### CODE REVIEW

File: plugins/inputs/postfix/README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
 
 ```toml @sample.conf
 # Measure postfix queue statistics
+# This plugin ONLY supports non-Windows OSes
```
Comment (srebhan) [inline plugins/inputs/postfix/README.md:23]: I think you used this below...
```suggestion
# This plugin ONLY supports non-Windows
```

**Review by srebhan — COMMENTED** [review]
Thanks for walking through this! Just one comment to make the test for "non-Windows" consistent.

**Comment by powersj** [comment]
Good catch! Updated

File: plugins/inputs/postfix/sample.conf [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # Measure postfix queue statistics
+# This plugin ONLY supports non-Windows OSes
```
Comment (powersj) [inline plugins/inputs/postfix/sample.conf:2]:
```suggestion
# This plugin ONLY supports non-Windows
```

**Review by srebhan — APPROVED** [review]
Thanks for your effort @powersj!

**Pull request merged**